examples/dreambooth_custom_character/README.md

```diff
@@ -0,0 +1,200 @@
+
+While improvements in prompting can mitigate this to some extent, it is almost always a better bet to fine-tune the model with your own data. Here are a few reasons why: 
+
+1. No amount of clever prompting can lead the neural network to generate things it hasn’t seen enough of in its training data. We’re pretty confident there wasn’t a preponderance of fluffy cartoon oxen in [Stable Diffusion’s training data](https://laion.ai/blog/laion-5b/), which leads to the lackluster baseline results we’ll show below. It’s much easier for us to provide this information to the model directly than to wait and hope that a future version of Stable Diffusion will somehow find and include precisely the data we’re looking for.
+2. If you train or fine-tune open source models with your own data rather than remaining beholden to third-party services, you can embed the model directly into your application. This will likely save you money over purchasing credits to existing providers, and eliminate your dependency on a vendor that could change its API / pricing or stop operating at any time.
+3. Fine-tuning with your own proprietary data allows you to gain a competitive advantage that you can’t access with prompting alone. If your IP relies solely on the specific way you prompt a publicly accessible model, it’s much easier for competitors to replicate your results!
+
+At [Oxen](https://oxen.ai/), we’re firm believers in the power of enhancing foundational models with domain-specific data. In this tutorial, we’ll use an Oxen repository and the [Dreambooth](https://dreambooth.github.io/) fine-tuning technique to create a Stable Diffusion variant that can consistently reproduce cute, fluffy cartoon oxen like the one currently hanging out [on our 404 page](https://www.oxen.ai/not-a-real-url).
+
+![The 404 Not Found page for oxen.ai, showing a cute cartoon ox](images/404.png)
+
+Here's a quick preview of the results we'll be able to achieve with only a few minutes of training on only six images.
+
+
+
+
+**Before Dreambooth:**
+![4 generated images under the prompt "an image of a happy white fluffy cartoon ox eating almonds", showing limited coherence / consistency](images/happy-fluffy-before.png)
+![4 generated images under the prompt "the oxenai ox eating almonds", showing limited coherence / consistency](images/oxenai-ox-before.png)
+
+**After Dreambooth:**
+![4 generated images under the same "fluffy cartoon ox" prompt as above, but more stylistically coherent](images/happy-fluffy-after.png)
+![4 generated images under the same "oxenai ox" prompt as above, but more stylistically coherent](images/oxenai-ox-after.png)
+
+These are some pretty exciting results, both in the increased coherence and consistency of the generated images, and the ease with which we could teach the model to recognize "the oxenai ox" as a specific token. Let's get started!
```

Comment:
I think we need to introduce the concept of a specific token you are fine tuning the model with before we mention the "oxenai ox". Just a quick title and blurb on how fine-tuning a token works and why we would do it.